### PR TITLE
FW release 2.8.1

### PR DESCRIPTION
**Pull request by matejcik**
Also adding a script that can generate releases.json -- sorely needed!
It's still kind of incomplete though.

### CODE REVIEW

**Comment by matejcik** [comment]
tagging @mmilata to review the script, but probably no need to wait for his review. we can modify the script later if there are problems.

File: firmware/t2t1/releases.json
Comment (komret) [inline firmware/t2t1/releases.json:1]: Please add `firmware_revision: 632b9561559b7ab6824bb7eeac072874e07b7b82` for T2T1. We use it for a new security check: https://github.com/trezor/trezor-suite/pull/13495

We'll eventually add it for all models.

File: scripts/releases-json.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+import json
+import subprocess
+import re
+import typing as t
+from pathlib import Path
+
+import click
+from trezorlib import firmware
+
+ROOT = Path(__file__).parent.parent.resolve()
```
Comment (komret) [inline scripts/releases-json.py:61]: The URLs should start with `data/firmware/`, but they start with `firmware/`

**Review by komret — CHANGES_REQUESTED** [review]
How about folders `1` and `2`? If we stop updating them, we need to adjust the [releases check](https://github.com/trezor/data/actions/runs/10316751076/job/28559620771?pr=91).

File: firmware/t2t1/releases.json
Comment (matejcik) [reply firmware/t2t1/releases.json:1]: added

File: scripts/releases-json.py (hunk 1 above)
Comment (matejcik) [reply scripts/releases-json.py:61]: fixed

**Comment by matejcik** [comment]
changed in force-push (messed up my fixups, sorry.)

**Review by komret — APPROVED** [review]

**Review by vdovhanych — APPROVED** [review]

**Pull request merged**